src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/model/sar/SARConditionFormatter.kt

```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.digital.hmpps.hmppshdcapi.model.sar
+
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.conditions.ConditionMetadata
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.conditions.V1_CONDITIONS
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.conditions.V2_CONDITIONS
+
+object SARConditionFormatter {
+
+  val policyVersions = mapOf(1 to V1_CONDITIONS, 2 to V2_CONDITIONS)
+
+  fun Map<Int, List<ConditionMetadata>>.getConditionText(conditionVersion: Int?, id: String) = policyVersions[conditionVersion]?.find { it.id == id }?.text ?: id
+
+  fun Map<Int, List<ConditionMetadata>>.getConditionText(id: String) = policyVersions.values.flatMap { it.filter { condition -> condition.id == id } }.takeIf { it.size == 1 }
+    ?.firstOrNull()?.text ?: id
```

Comment:
The licence versions we won't have additional condition version so we need to guess what policy version they are using based on the presence of expected conditions.

There are a couple of conditions ids that appear on both policy - if it's one of these we can't easily guess which policy version to use so just fall back to the licence id rather than condition text.

We could possibly make this more robust by determining version based on the presence of any unique condition on a policy version and then use that for all conditions - this would still present a problem for cases where they only choose conditions that are indeterminate.